.gitignore

```diff
@@ -26,6 +26,7 @@ dist-ssr
 # Supabase
 .branches
 .temp
+supabase/functions/**/.env
```

Comment:
To improve maintainability and streamline deployments, I recommend managing all environment variables directly within the supabase folder. This helps prevent scattering variables across the project, making it easier to aggregate them during deployment or CI/CD processes.